Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/docrepository/adapter/model/DocumentMetadata.java

```diff
@@ -0,0 +1,677 @@
+    }
+
+    public String getLegalAuthenticator() {
+        return legalAuthenticator;
+    }
+
+    public void setLegalAuthenticator(String legalAuthenticator) {
+        this.legalAuthenticator = legalAuthenticator;
+    }
+
+    public String getMimeType() {
+        return mimeType;
+    }
+
+    public void setMimeType(String mimeType) {
+        this.mimeType = mimeType;
+    }
+
+    public String getParentDocumentId() {
+        return parentDocumentId;
+    }
+
+    public void setParentDocumentId(String parentDocumentId) {
+        this.parentDocumentId = parentDocumentId;
+    }
+
+    public String getParentDocumentRelationship() {
+        return parentDocumentRelationship;
+    }
+
+    public void setParentDocumentRelationship(String parentDocumentRelationship) {
+        this.parentDocumentRelationship = parentDocumentRelationship;
+    }
+
+    public String getPatientId() {
+        return patientId;
+    }
+
+    public void setPatientId(String patientId) {
+        this.patientId = patientId;
+    }
+
+    public String getPid11() {
+        return pid11;
+    }
+
+    public void setPid11(String pid11) {
+        this.pid11 = pid11;
+    }
+
+    public String getPid3() {
+        return pid3;
+    }
+
+    public void setPid3(String pid3) {
+        this.pid3 = pid3;
+    }
+
+    public String getPid5() {
+        return pid5;
+    }
+
+    public void setPid5(String pid5) {
+        this.pid5 = pid5;
+    }
+
+    public String getPid7() {
+        return pid7;
+    }
+
+    public void setPid7(String pid7) {
+        this.pid7 = pid7;
+    }
+
+    public String getPid8() {
+        return pid8;
+    }
+
+    public void setPid8(String pid8) {
+        this.pid8 = pid8;
+    }
+
+    public String getPracticeSetting() {
+        return practiceSetting;
+    }
+
+    public void setPracticeSetting(String practiceSetting) {
+        this.practiceSetting = practiceSetting;
+    }
+
+    public String getPracticeSettingDisplayName() {
+        return practiceSettingDisplayName;
+    }
+
+    public void setPracticeSettingDisplayName(String practiceSettingDisplayName) {
+        this.practiceSettingDisplayName = practiceSettingDisplayName;
+    }
+
+    public String getPracticeSettingScheme() {
+        return practiceSettingScheme;
+    }
+
+    public void setPracticeSettingScheme(String practiceSettingScheme) {
+        this.practiceSettingScheme = practiceSettingScheme;
+    }
+
+    public Integer getSize() {
+        return size;
+    }
+
+    public void setSize(Integer size) {
+        this.size = size;
+    }
+
+    public String getSourcePatientId() {
+        return sourcePatientId;
+    }
+
+    public void setSourcePatientId(String sourcePatientId) {
+        this.sourcePatientId = sourcePatientId;
+    }
+
+    public Date getCreationTime() {
+        if (creationTime == null) {
+            return null;
+        } else {
+            return (Date) creationTime.clone();
```

Comment:
Why do we clone creationTime but not other?

Reply:
No Idea. See above comment. I renamed the class, and Git thinks I created the entire file. I left it the way it was.